Package.swift

```diff
@@ -5,24 +5,21 @@ import PackageDescription
 
 let package = Package(
     name: "iosDesignTokens",
+    platforms: [.iOS(.v14)],
     products: [
         // Products define the executables and libraries a package produces, and make them visible to other packages.
         .library(
             name: "iosDesignTokens",
             targets: ["iosDesignTokens"]),
     ],
-    dependencies: [
-        // Dependencies declare other packages that this package depends on.
-        // .package(url: /* package url */, from: "1.0.0"),
-    ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages this package depends on.
         .target(
             name: "iosDesignTokens",
             dependencies: []),
-        .testTarget(
-            name: "iosDesignTokensTests",
-            dependencies: ["iosDesignTokens"]),
+//        .testTarget(
+//            name: "iosDesignTokensTests",
+//            dependencies: ["iosDesignTokens"]),
```

Comment:
These left as comments?

Reply:
I kept it in case we ever wanted tests but I can remove it